### PR TITLE
React framework

### PR DESCRIPTION
**Pull request by herecomesjaycee**
Reference: [Blog](https://mherman.org/blog/dockerizing-a-react-app/)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# base image
+FROM node:12.2.0-alpine
```
Comment (GenesisSelf) [inline Dockerfile:2]: Do we want to use node 12?

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
-# period-dignity-app-client
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (GenesisSelf) [inline README.md:1]: Loving how meticulous this is 💃

File: public/robots.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (GenesisSelf) [inline public/robots.txt]: Do we need this file?

**Review by GenesisSelf — COMMENTED** [review]
Just a couple of questions regarding some files and versions.

**Review by GenesisSelf — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (mjg123) [reply Dockerfile:2]: Discussed with team - I think it's good unless we find a reason not to.

File: public/robots.txt (hunk 3 above)
Comment (mjg123) [reply public/robots.txt]: I'll remove it

**Pull request merged**